std/numeric.d

```diff
@@ -2572,27 +2572,55 @@ GapWeightedSimilarityIncremental!(R, F) gapWeightedSimilarityIncremental(R, F)
 
 /**
 Computes the greatest common divisor of $(D a) and $(D b) by using
-Euclid's algorithm.
+Euclid's or Stein's algorithm depending on compiler.
```

Comment:
Oh, you may also add Web links to the algorithms:  "... using an efficient algorithm such as $(HTTP xyz.com/page.html, Euclid) or $(HTTP xyz.com/page.html, Stein)."